images/cruise-control/Dockerfile

```
@@ -0,0 +1,35 @@
+FROM openjdk:8-jdk as build-env
+ARG CRUISE_CONTROL_VERSION
+ARG CRUISE_CONTROL_UI_VERSION
+RUN git clone --branch ${CRUISE_CONTROL_VERSION} https://github.com/linkedin/cruise-control.git && cd cruise-control/
+WORKDIR cruise-control
+RUN ./gradlew jar --warning-mode=all
+RUN wget https://github.com/linkedin/cruise-control-ui/releases/download/v${CRUISE_CONTROL_UI_VERSION}/cruise-control-ui-${CRUISE_CONTROL_UI_VERSION}.tar.gz && mkdir /cruise-control-ui && \
+    tar -xvvf cruise-control-ui-${CRUISE_CONTROL_UI_VERSION}.tar.gz --strip-components=1 -C /cruise-control-ui && rm cruise-control-ui-${CRUISE_CONTROL_UI_VERSION}.tar.gz
+RUN wget https://downloads.mesosphere.com/java/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz -P /srv && \
+    mkdir /srv/jdk && \
+    tar -xvvf /srv/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz --strip-components=1 -C /srv/jdk
+
+# using debian:stretch-20190610-slim
+FROM debian@sha256:9490c476443a3869e39c2897fa66c91daf5dcbbfca53c976dac7bbdc45775b28
+
+RUN apt-get update && apt-get install -y curl openssl netcat procps
+RUN rm -rf /var/lib/apt/lists/*
```

```suggestion
    rm -rf /var/lib/apt/lists/*
```
This needs to be in the same layer as the `apt-get update` to be beneficial.